docs/zh_cn/design/logging.md

```diff
@@ -412,39 +412,42 @@ logger = MMLogger.get_instance('mmengine', log_file='tmp.log', distributed=True,
 
 ```text
 #  共享存储
-./tmp
-├── tmp.log
-├── tmp_rank1.log
-├── tmp_rank2.log
-├── tmp_rank3.log
-├── tmp_rank4.log
-├── tmp_rank5.log
-├── tmp_rank6.log
-└── tmp_rank7.log
+work_dir/20230228_141908
+├── 20230306_183634_${hostname}_device0_rank0.log
+├── 20230306_183634_${hostname}_device1_rank1.log
+├── 20230306_183634_${hostname}_device2_rank2.log
+├── 20230306_183634_${hostname}_device3_rank3.log
+├── 20230306_183634_${hostname}_device4_rank4.log
+├── 20230306_183634_${hostname}_device5_rank5.log
+├── 20230306_183634_${hostname}_device6_rank6.log
+├── 20230306_183634_${hostname}_device7_rank7.log
 ...
-└── tmp_rank63.log
+├── 20230306_183634_${hostname}_device7_rank63.log
 ```
 
 多机多卡，独立存储的情况：
 
 ```text
 # 独立存储
 # 设备0：
-work_dir/
-└── exp_name_logs
-    ├── exp_name.log
-    ├── exp_name_rank1.log
-    ├── exp_name_rank2.log
-    ├── exp_name_rank3.log
-    ...
-    └── exp_name_rank7.log
+work_dir/20230228_141908
+├── 20230306_183634_${hostname}_device0_rank0.log
+├── 20230306_183634_${hostname}_device1_rank1.log
+├── 20230306_183634_${hostname}_device2_rank2.log
+├── 20230306_183634_${hostname}_device3_rank3.log
+├── 20230306_183634_${hostname}_device4_rank4.log
+├── 20230306_183634_${hostname}_device5_rank5.log
+├── 20230306_183634_${hostname}_device6_rank6.log
+├── 20230306_183634_${hostname}_device7_rank7.log
 
 # 设备7：
-work_dir/
-└── exp_name_logs
-    ├── exp_name_rank56.log
-    ├── exp_name_rank57.log
-    ├── exp_name_rank58.log
-    ...
-    └── exp_name_rank63.log
+work_dir/20230228_141908
+├── 20230306_183634_${hostname}_device0_rank0.log
+├── 20230306_183634_${hostname}_device1_rank1.log
+├── 20230306_183634_${hostname}_device2_rank2.log
+├── 20230306_183634_${hostname}_device3_rank3.log
+├── 20230306_183634_${hostname}_device4_rank4.log
+├── 20230306_183634_${hostname}_device5_rank5.log
+├── 20230306_183634_${hostname}_device6_rank6.log
+├── 20230306_183634_${hostname}_device7_rank7.log
```

Comment:
Even if the storage is non-shared, the rank id is still the global rank. The distributed group has no relation with the storage.